src/components/ContactTop/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { charityAPI } from '../../clients/charity';
+class ContactTop extends Component {
+  state = {
+    soicilIconsList: [],
+    loading: false,
+    error: false
+  };
+
+  componentDidMount() {
+    charityAPI('/socialmedias')
+      .then(({ data: soicilIconsList }) => {
+        this.setState({ soicilIconsList });
+      })
+      .catch(error => {});
+  }
+  render() {
+    if (this.state.loading) {
+      return <div>loading data </div>;
+    }
+
+    if (this.state.error) {
+      return <div>we can not fetch data</div>;
```

Comment:
You will never reach this code.
I'll leave it to you to know why 😉

Reply:
Maybe because I did not change the state of error inside the catch method 

```
  componentDidMount() {
    charityAPI('/socialmedias')
      .then(({ data: soicilIconsList }) => {
        this.setState({ soicilIconsList });
      })
      .catch(error => {
        this.setState({
          error:true,
        })
      });
  }
```